Code/Framework/AzFramework/AzFramework/Terrain/TerrainDataRequestBus.h

```diff
@@ -130,6 +130,27 @@ namespace AzFramework
                 SurfaceData::SurfacePoint& outSurfacePoint,
                 Sampler sampleFilter = Sampler::DEFAULT,
                 bool* terrainExistsPtr = nullptr) const = 0;
+
+        private:
+            // Private variations of the GetSurfacePoint API exposed to BehaviorContext that returns a value instead of
+            // using an "out" parameter.  The "out" parameter is useful for reusing memory allocated in SurfacePoint when
```

Comment:
uber-nit: double space.
```
// using an "out" parameter. The "out" parameter
```

Reply:
Lol. I'm from the old-school style guides that require 2 spaces after a period, but I will concede that as of 2019, all major style guides are now in agreement on single space, and as of 2020, MS Word marks two spaces as an error. I'll change it.

Reply:
It took a while to break this habit for me too. I feel so betrayed by the 90's 😞 .